Source/WebCore/mathml/MathMLSelectElement.cpp

```diff
@@ -164,7 +164,7 @@ Element* MathMLSelectElement::getSelectedSemanticsChild()
     if (!child)
         return nullptr;
 
-    if (!is<MathMLElement>(*child) || !downcast<MathMLElement>(*child).isPresentationMathML()) {
+    if (!dynamicDowncast<MathMLElement>(*child)->isPresentationMathML()) {
```

Comment:
`dynamicDowncast` might return null when child is not `MathMLElement`, and this will be null deference

Reply:
This can be better written as:
```
if (auto* childElement = dynamicDowncast<MathMLElement>(*child); childElement && childElement->isPresentationMathML()) {
```

This is what Sihui meant and she's right. It avoid doing the type check twice.